module/hooks/action.mjs

```diff
@@ -548,6 +566,7 @@ HOOKS.poisonIngest = {
 
 HOOKS.rallyingCry = {
   postActivate(outcome) {
+    if ( outcome.self ) return;
```

Comment:
With my change to rallying cry being a heroism spender, I had envisioned that it would now apply to yourself as well.